setup.py

```diff
@@ -1,39 +1,47 @@
-from setuptools import setup, find_packages
-import sys, os
+from setuptools import find_packages
+from setuptools import setup
+
+import os
+
 
 def read(*rnames):
     return open(os.path.join(os.path.dirname(__file__), *rnames)).read()
 
+
 long_description = read('src/z3c/saconfig/README.rst') + '\n' + read('CHANGES.rst')
 
 setup(name='z3c.saconfig',
       version='0.15.dev0',
       description="Minimal SQLAlchemy ORM session configuration for Zope",
       long_description=long_description,
-      # Get more strings from http://www.python.org/pypi?%3Aaction=list_classifiers
+      # Get more strings from https://pypi.org/pypi?%3Aaction=list_classifiers
       classifiers=['Development Status :: 4 - Beta',
                    'Environment :: Web Environment',
                    'Framework :: Zope :: 3',
+                   'Framework :: Zope :: 4',
                    'Intended Audience :: Developers',
                    'License :: OSI Approved :: Zope Public License',
                    'Operating System :: OS Independent',
                    'Programming Language :: Python :: 2.7',
+                   'Programming Language :: Python :: 3.5',
+                   'Programming Language :: Python :: 3.6',
                    'Topic :: Internet :: WWW/HTTP',
                    'Topic :: Software Development :: Libraries',
                    'Topic :: Database',
                    ],
       keywords='zope relational sqlalchemy component integration',
       author='Martijn Faassen',
       author_email='faassen@startifact.com',
-      url='http://pypi.python.org/pypi/z3c.saconfig',
+      url='https://pypi.org/project/z3c.saconfig/',
```

Comment:
IMHO a GitHub link would be better -- this is the URL shown on PyPI, and it's a little pointless to have a PyPI page point back to itself.